validator/webui/index.html

```diff
@@ -456,6 +456,11 @@
           button.addEventListener('click', function() {
             var urlToFetch = input.value.trim();
             if (urlToFetch.length > 0) {
+              if (!(urlToFetch.indexOf('http://') == 0 ||
+                    urlToFetch.indexOf('https://') == 0)) {
+                urlToFetch = 'https://' + urlToFetch;
```

Comment:
I disagree, this should be as easy to use as possible to validate amp pages. Just because I'm putting them up as http for now doesn't mean they won't be https later. Lets not put up barriers for those wanting to develop AMP HTML.